### PR TITLE
Use index of job config in the Copr source script

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to packit/packit#1493

---
N/A

### CODE REVIEW

**Comment by lbarcziova** [comment]
recheck

**Comment by lbarcziova** [comment]
recheck

**Review by TomasTomecek — APPROVED** [review]
wanted to verify locally this works as expected

and it does!

```
In [4]: j1 = JobConfig(type=JobType.build, trigger=JobConfigTriggerType.release)

In [5]: j2 = JobConfig(type=JobType.build, trigger=JobConfigTriggerType.pull_request)

In [6]: j3 = JobConfig(type=JobType.tests, trigger=JobConfigTriggerType.pull_request)

In [7]: jobs = [j1, j2, j3]

In [8]: jobs.index(j1)
Out[8]: 0

In [9]: jobs.index(j3)
Out[9]: 2
```

thanks for such a quick fix!

**Comment by lbarcziova** [comment]
recheck

File: packit_service/worker/build/copr_build.py [hunk 1]
```diff
@@ -345,6 +345,13 @@ def get_packit_copr_download_urls(self) -> List[str]:
             )
             raise ex
 
+    def get_job_config_index(self):
```
Comment (TomasTomecek) [inline packit_service/worker/build/copr_build.py:348]:
```suggestion
    def get_job_config_index(self) -> int:
```
just to be clear it's indeed int :)

File: tests/unit/test_copr_build.py [hunk 2]
```diff
@@ -2113,3 +2113,73 @@ def test_get_packit_copr_download_urls(github_pr_event):
...
+        CoprBuildJobHelper(
+            package_config=package_config,
+            job_config=job_config,
+            service_config=None,
+            project=None,
+            metadata=None,
+            db_trigger=None,
+        ).get_job_config_index()
+        == result
+    )
```
Comment (TomasTomecek) [inline tests/unit/test_copr_build.py:2185]: very nice and thorough!

**Review by TomasTomecek — APPROVED** [review]

**Comment by lbarcziova** [comment]
recheck

**Pull request merged**